crypto/dsa/dsa_ossl.c

```diff
@@ -95,7 +95,7 @@ DSA_SIG *dsa_do_sign_int(OPENSSL_CTX *libctx, const unsigned char *dgst,
     blind = BN_CTX_get(ctx);
     blindm = BN_CTX_get(ctx);
     tmp = BN_CTX_get(ctx);
-    if (tmp == NULL)
+    if (m == NULL || tmp == NULL)
```

Comment:
This is not necessary. You, by design, only need to check the last call the `BN_CTX_get()`

Reply:
I am aware of this. Coverity however I dont think knows this, so it may be thinking that tmp is 'possibly' NULL. I can mark it as a false positive if you prefer - but thought it wasnt a big problem adding the extra check..

Reply:
Mark it as a false positive.  Adding the extra check will mean that future people reading the code will not understand the semantics properly.  We already have enough problems with that, as this diff shows :)

Reply:
Since we now use travis_terminate, this is less important that it used to be, but no harm in keeping it as-is.

Reply:
I agree. This should be marked as a false positive.

Reply:
Done - rebased to update the commit + updated coverity database comment